vars/virtualenv.groovy

```diff
@@ -1,16 +1,8 @@
-def create(String python, String destDir) {
-  sh("virtualenv --python=${python} ${destDir}")
-}
-
-
-def installRequirements(String destDir) {
-  run(destDir, "pip install -r requirements.txt")
-}
+import com.ableton.VirtualEnv
 
 
-def run(String destDir, String command) {
-  sh("""
-    . ./${destDir}/bin/activate
-    ${command}
-  """)
+def create(script, String python) {
```

Comment:
`WorkflowScript script` then :wink:

Reply:
Ok sadly using a type for `WorkflowScript` doesn't work (see http://newci.office.ableton.com:8080/job/AbletonDevTools-runthebuilds/48/console). So we're back to just `def` for script. 😿